packages/logging/src/log.js

```diff
@@ -520,8 +526,6 @@ Log.prototype.warning = function(entry, options, callback) {
  * @param {object[]} options.labels - Labels to set on the log.
  * @param {object} options.resource - A default monitored resource for entries
  *     where one isn't specified.
- * @param {boolean} options.removeCircular - Replace circular references in an
```

Comment:
That's do-able, but I don't think there is a strong use-case for this. It will be marginally unsightly: we'll have to remove this property before adding `options` to `reqOpts`.